### ISSUE TITLE
Compact tree for branches from same base commit

### ISSUE THREAD

**Issue by Adarma** [SublimeTextIssues/Merge#17]
I have many branches with the same base commit. In the commandline it is drawn like this:

![image](https://user-images.githubusercontent.com/4405072/45879003-4208ec00-bda3-11e8-89d5-f487bf0d21ee.png)

It does not take up much horizontal space as each branch is only indented one level.

In Sublime Merge, it looks like this:

![image](https://user-images.githubusercontent.com/4405072/45879064-6c5aa980-bda3-11e8-90af-59adbaa50f98.png)

It takes up a lot of horizontal space as each branch is indented one level more than the last.

Could a more compact tree be drawn in the commandline style?

**Comment by FichteFoll** [comment]
FYI, as of 1067, merge commits are now collapsed by default and can be expanded with a click on the ... square. Not sure if there is a key binding for it since right arrow key still expands the changed files list for those.

That doesn't help with unmerged branches, however.

**Comment by Adarma** [comment]
Yeah, that does not really address this. It is the unmerged branches that are the issue.
I created an [issue](https://github.com/sublimehq/sublime_merge/issues/19) on the SublimeHQ sublime_merge github site once that was announced.
